sources/packages/forms/src/form-definitions/sfaa2025-26.json

```diff
@@ -10918,7 +11090,7 @@
           },
           "properties": {},
           "lockKey": true,
-          "customConditional": "show = data.dependantStatus === \"independant\";"
+          "customConditional": "show = !!data.dependantstatus;"
```

Comment:
Thanks for change. This must be javascript condition and no changes required here.